pkg/importer/spanner/import_test.arrai

```diff
@@ -0,0 +1,3 @@
+let importlib = //{./import};
```

Comment:
Files ending in `_test.arrai` will be treated as unit test specs by `arrai test`. They should not expect any `//os.args`, and should return only `true` or a data structure full of `true` values.

Reply:
will rename